apex/contrib/csrc/xentropy/interface.cpp

```diff
@@ -0,0 +1,56 @@
+#include <pybind11/pybind11.h>
+#include <pybind11/numpy.h>
+#include <pybind11/stl.h>
```

Comment:
I don't think these lines are necessary.  <torch/extension.h> should be sufficient.